src/routeGeneration/templates/hapi.ts

```diff
@@ -41,15 +41,16 @@ export function RegisterRoutes(server: any) {
         server.route({
             method: '{{method}}',
             path: '{{fullPath}}',
-            config: {
+            options: {
                 {{#if security.length}}
                 pre: [
                     {
-                      method: authenticateMiddleware({{json security}})
+                      method: authenticateMiddleware({{json security}}),
+                      assign: "user"
                     }
                 ],
                 {{/if}}
-                handler: (request: any, reply: any) => {
+                handler: (request: any, h: any) => {
```

Comment:
what does `h` stand for?

Reply:
I asked myself the same one and the only match I could find is Hapi ... just a guess.
@slapersonne ?